cmd/kubeadm/app/cmd/phases/join/controlplaneprepare.go

```diff
@@ -66,6 +74,20 @@ func getControlPlanePreparePhaseFlags() []string {
 		options.TokenDiscovery,
 		options.TokenDiscoveryCAHash,
 		options.TokenDiscoverySkipCAHash,
+		options.CertificateKey,
+	}
+}
+
+func newControlPlanePrepareDownloadCertsSubphase() workflow.Phase {
+	return workflow.Phase{
+		Name:  "download-certs",
+		Short: fmt.Sprintf("Download certificates from %s", kubeadmconstants.KubeadmCertsSecret),
+		Long:  cmdutil.MacroCommandLongDescription,
```

Comment:
You don't need the `MacroCommandLongDescription` here. There are no sub-phases to this one.